docker-compose.prod.yml

```diff
@@ -0,0 +1,39 @@
+services:
+  db:
+    image: postgres:16.4
+    restart: unless-stopped
+    volumes:
+      - ${LOCAL_WORKSPACE_FOLDER:-.}/data/db:/var/lib/postgresql/data
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready -U postgres"]
+      interval: 3s
+      timeout: 3s
+      retries: 5
+    env_file: ./server/.env
+    ports:
+      - 5432:5432
+  server:
+    container_name: wajo-server
+    build:
+      context: .
+      dockerfile: ./docker/server/Dockerfile
```

Comment:
no... DO NOT build the docker image in the VPS...

Get it to pull from a docker image

Reply:
the reason why you don't do that is:
1. you want replicability by getting what worked in the artifact
2. building docker image requires lots of resource that your vps server may not have more
3. you'll want something like watchtower to do continuous deployment (which it relies on it coming from artifact store such as docker registry)